prov/efa/src/rdm/efa_rdm_ep_progress.c

```diff
@@ -563,7 +568,13 @@ ssize_t efa_rdm_ep_send_queued_pkts(struct efa_rdm_ep *ep,
 		 */
 		dlist_remove(&pkt_entry->entry);
 
-		ret = efa_rdm_pke_sendv(&pkt_entry, 1);
+		base_hdr = efa_rdm_pke_get_base_hdr(pkt_entry);
+		if (base_hdr->type == EFA_RDM_RMA_CONTEXT_PKT) {
+				ret = efa_rdm_pke_write(pkt_entry);
```

Comment:
The indentation seems too large. please keep it consistent with other if blocks.

Reply:
should we use `EFA_RDM_RDMA_WRITE_CONTEXT` here ?

Reply:
Fixed indentation

To get the context type, we need to cast to RMA packet. I added a debug assert for `EFA_RDM_RDMA_WRITE_CONTEXT`

Reply:
thanks! LGTM